app/src/main/java/com/eveningoutpost/dexdrip/cgm/carelinkfollow/message/TextMap.java

```diff
@@ -0,0 +1,345 @@
+package com.eveningoutpost.dexdrip.cgm.carelinkfollow.message;
+
+import java.util.HashMap;
+
+public class TextMap {
+
+
+    public static final String ERROR_TEXT_PREFIX_GUARDIAN = "GM_";
+    public static final String ERROR_TEXT_PREFIX_NGP = "N";
+
+    private static HashMap<String, String> errorTextMap;
+    private static HashMap<String, String> errorCodeMap;
+
+    static {
+
```

Comment:
Maybe the use of [immutable maps](https://docs.oracle.com/javase/9/docs/api/java/util/Map.html#ofEntries-java.util.Map.Entry...-) is more readable and appropriate here.

Reply:
Are you sure, that ofEntries is supported by Android?

Reply:
I have checked and it seems to me, that the more readable and appropriate `Map.of` and `Map.ofEntries` requires API level 30, but the compile SDK level is currently API 29. Or am I missing something?